python/mlcroissant/mlcroissant/_src/operation_graph/operations/read.py

```diff
@@ -191,6 +188,27 @@ def _read_file_content(
                     return pd.DataFrame({
                         FileProperty.content: [out],
                     })
+                elif encoding_format in {EncodingFormat.JPG, EncodingFormat.PNG}:
+                    try:
+                        img = deps.PIL_Image.open(file).convert("RGB")
+                    except ModuleNotFoundError:
+                        raise NotImplementedError(
+                            "Pillow is not installed and is a dependency"
+                        )
+                    return pd.DataFrame({FileProperty.content: [img]})
+                elif encoding_format == EncodingFormat.TIF:
+                    try:
+                        arr = deps.tifffile.imread(file)
+                        arr_min, arr_max = arr.min(), arr.max()
+                        arr_norm = (arr - arr_min) / (arr_max - arr_min)
+                        pil_img = deps.PIL_Image.fromarray(
+                            (arr_norm * 255).astype("uint8")
+                        )
+                    except ModuleNotFoundError:
+                        raise NotImplementedError(
+                            "Pillow or tifffile is not installed and is a dependency"
```

Comment:
this error message reads strange, can you fix it? You could also give a more actionable fix, like `Please run `pip install ...`